docs/src/user_docs/assembly/overview.md

```diff
@@ -1,22 +0,0 @@
-## Miden VM overview
-Miden VM is a stack machine. The base data type of the MV is a field element in a 64-bit [prime field](https://en.wikipedia.org/wiki/Finite_field) defined by modulus $p = 2^{64} - 2^{32} + 1$. This means that all values that the VM operates with are field elements in this field (i.e., values between $0$ and $2^{64} - 2^{32}$, both inclusive).
```

Comment:
do we want to remove this?

Reply:
This file is just a leftover and isn't linked anywhere

Reply:
ah, I thought it was https://github.com/0xPolygonMiden/miden-vm/blob/main/docs/src/intro/overview.md , thanks!